### DISCUSSION TITLE
How to return to the usual form of notifications?

### DISCUSSION THREAD

**Discussion by Kushiko** [prasanthrangan/hyprdots#2042] [Q&A] [answered]
Hello. At one point my notifications became square, I don't know why this happened.... How can I get it back?
![изображение](https://github.com/user-attachments/assets/f3203731-8d5e-4252-9f1a-e84639cffe87)

**Comment by kRHYME7** [comment, marked as answer]
check the Pull Request.
Use 
`height= (0,300)`  in dunst.conf then change themes to generate the dunstrc

instead of `height= 300`
